group_vars/location_f17/networks.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+# 10.31.167.192/27
+#	- 10.31.167.192/28 - 10.31.167.207/28  (management network)
+#	- 10.31.167.208/28 - 10.31.167.223/28  (for announcing)
+# 2001:bf7:830:b000::/56
+#	- 2001:bf7:830:b000::/57               (management network)
+#	- 2001:bf7:830:b080::/57               (for announcing)
+
+# DHCP:
+# 10.36.189.0/24
+# 2001:bf7:830:6600::/56
+
+ipv6_prefix: "2001:bf7:830:b000::/56"
+
+networks:
+  - vid: 6
+    role: mesh
+    name: mesh_emma
+    prefix: 10.31.167.209/32
```

Comment:
You can start with `10.31.167.208/32` as ip from your mesh-pool. :)

Reply:
done!